src/payloads/services/special.ts

```diff
@@ -53,14 +53,34 @@ const withCgnService = (
   organization_fiscal_code: organizationFiscalCode
 });
 
+export const cdcServiceId = "01G2AFTME08TS0QD2P2S682CJ0";
+const cdcService: ServicePublic = {
+  ...getService(cdcServiceId),
+  organization_name: "Ministero beni culturali" as OrganizationName,
+  service_name: "Carta Della Cultura" as ServiceName,
+  service_metadata: {
+    ...getServiceMetadata(ServiceScopeEnum.NATIONAL),
+    category: SpecialServiceCategoryEnum.SPECIAL,
+    custom_special_flow: "cdc" as SpecialServiceMetadata["custom_special_flow"]
+  }
+};
+
+const withCdcService = (
+  organizationFiscalCode: OrganizationFiscalCode
+): ServicePublic => ({
+  ...cdcService,
+  organization_fiscal_code: organizationFiscalCode
+});
+
 // list of tuple where the first element is a flag indicating if the relative service should be included
 // the second element is the service factory
 const specialServicesFactory: ReadonlyArray<readonly [
   boolean,
   (organizationFiscalCode: OrganizationFiscalCode) => ServicePublic
 ]> = [
   [ioDevServerConfig.services.includeSiciliaVola, withSiciliaVolaService],
-  [ioDevServerConfig.services.includeCgn, withCgnService]
+  [ioDevServerConfig.services.includeCgn, withCgnService],
+  [true, withCdcService]
```

Comment:
```suggestion
  [true, withCdcService]
```
Could be a good idea update the config by including a flag for CDC service too?